cpp_utils/include/cpp_utils/ros2_mangling.hpp

```diff
@@ -0,0 +1,220 @@
+// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <cpp_utils/library/library_dll.h>
+
+namespace eprosima {
+namespace utils {
+
+/**
+ * @param[in] name string that will be stripped from prefix
+ * @param[in] prefix prefix to be stripped
+ * @return name stripped of prefix, or
+ * @return "" if name doesn't start with prefix
+ */
+CPP_UTILS_DllAPI
+std::string remove_prefix(
+        const std::string& name,
+        const std::string& prefix);
+
+/**
+ * @param[in] name string to be prefixed with "prefix"
+ * @param[in] prefix prefix to be added
+ * @return Returns the name with the prefix added "prefixname"
+ */
+CPP_UTILS_DllAPI
+std::string add_prefix(
+        const std::string& name,
+        const std::string& prefix);
+
+/**
+ * @param[in] name string to be suffixed with "suffix"
+ * @param[in] suffix suffix to be added
+ * @return Returns the name with the suffix added "namesuffix"
+ */
+CPP_UTILS_DllAPI
+std::string add_suffix(
+        const std::string& name,
+        const std::string& suffix);
+
+/**
+ * @param[in] topic_name name of the topic
+ * @return Returns the ROS specific prefix if present, otherwise "".
+ */
+CPP_UTILS_DllAPI
+std::string get_ros_prefix_if_exists(
```

Comment:
Unused function?

Reply:
Reserved for future use, currently not utilized.